diagnostic.md

```diff
@@ -51,13 +62,20 @@ Write the `curl` request you'd use to delete a single donut, using whatever
 ID you'd like.
 
 ```sh
-# your answer here
-```
+curl --include --request DELETE http://www.example.com/donuts/1 \
+  --header "Authorization: Token token=$TOKEN"```
 
 Write an AJAX request to delete a single donut from the server.
 
 ```js
-let deleteDonut = /* your answer here */;
+let deleteDonut = function() {
+  return $.ajax({
+    url: http://www.example.com/donuts,
```

Comment:
You need to have the ID of the specific donut you're deleting here, similar to the "Get a single resource" example